swo_aws_extension/airtable/models.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass, field
+from enum import StrEnum
+from typing import Any, Self
+
+
+class OpScaleFields(StrEnum):
```

Comment:
the only thing I would like you to do is to get rid of `OpScale` word here. Cause

1. The solution is called `OptScale` :-)
2. And it is a concrete implementation of the FinOps solution.

So I would prefer to use everywhere - FinOps term instead of the OptScale